rest/kadai-rest-spring/src/main/java/io/kadai/task/rest/assembler/AttachmentRepresentationModelAssembler.java

```diff
@@ -65,7 +62,6 @@ public AttachmentRepresentationModel toModel(@NonNull Attachment attachment) {
     repModel.setObjectReference(objectReferenceAssembler.toModel(attachment.getObjectReference()));
     repModel.setChannel(attachment.getChannel());
     repModel.setCustomAttributes(attachment.getCustomAttributeMap());
-    repModel.add(linkTo(AttachmentController.class).slash(attachment.getId()).withSelfRel());
```

Comment:
Why did you delete this?

Reply:
Because I deleted the class AttachmentController, which is completely empty and without annotations